string ip_address   # IP address
string ip_mask      # IP mask
string ip_gateway   # IP gateway
---
